kong/tools/wrpc/utils.lua

```diff
@@ -0,0 +1,74 @@
+local semaphore = require "ngx.semaphore"
+
+local table_insert = table.insert     -- luacheck: ignore
```

Comment:
is `-- luacheck: ignore` necessary? Could we remove this comments?

Reply:
here `table.insert()` is only used in the "append" form, which is compiled to exactly the same IR as `t[#t+1] = x`  so it could just be avoided.